src/Hyperrectangle.jl

```diff
@@ -59,6 +59,7 @@ Hyperrectangle(center::Vector{N}, radius::Vector{N}) where {N<:Real} =
 function Hyperrectangle(;
                         high::AbstractVector{N},
                         low::AbstractVector{N}) where {N<:Real}
+    @assert all(low .<= high) "lower bound must be lower than upper bound"
```

Comment:
more efficient check:
```suggestion
    @assert all(i -> low[i] <= high[i], eachindex(low)) "lower bound must be lower than upper bound"
```